src/StructuredLogViewer.Core/NodeQueryMatcher.cs

```diff
@@ -21,6 +21,11 @@ public class NodeQueryMatcher
         public bool IncludeDuration { get; set; }
         public TimeSpan PrecalculationDuration { get; set; }
 
+        private bool searchInName { get; set; }
+        private bool searchInValue { get; set; }
+        private string nameToSearch { get; set; }

```

Comment:
Do we really need the two bool properties? We can just use the logic of whether the string is null or not. But we can fix next time.